Storyshots.test.js

```diff
@@ -58,14 +58,13 @@ function removeMaterialUIInternals(json) {
 
   return json;
 }
-
+MockDate.set('2020-01-01');
```

Comment:
Actually I am not sure if `test()` is called first, or if `MockDate.reset()` is called first. Depends on how `initStoryshots` calls `test()`.

Judging from `Info`'s snapshot change (`over 30 years` -> `about 31 years`) I suspect that the mock is not in effect.

Reply:
you're right, I didn't think it through lol
I've modified the test accordingly.